src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -171,25 +193,25 @@ const TableToolbar = ({
         {...search}
         id={`${tableId}-toolbar-search`}
         onChange={event => onApplySearch(event.currentTarget ? event.currentTarget.value : '')}
-        placeHolderText={searchPlaceholderText}
+        placeHolderText={i18n.searchPlaceholder}
         disabled={isDisabled}
       />
     ) : null}
     <StyledTableToolbarContent>
       {customToolbarContent || null}
       {totalFilters > 0 ? (
         <Button kind="secondary" onClick={onClearAllFilters}>
-          {clearAllFiltersText}
+          {i18n.clearAllFilters}
         </Button>
       ) : null}
       {hasColumnSelection ? (
         <ToolbarSVGWrapper onClick={onToggleColumnSelection}>
-          <IconColumnSelector description={columnSelectionText} />
+          <IconColumnSelector description={i18n.columnSelection} />
```

Comment:
@tay1orjones should this be `i18n.columnSelectionButtonAria`?

Reply:
Yep, great catch, thanks!